app/assets/javascripts/image.js

```diff
@@ -0,0 +1,47 @@
+function loadImages() {
+  var item = $('#image_uuid');
+  var url = item.data('url');
+  var selected = item.data('selected');
+
+  item.indicator_show();
+  foreman.tools.showSpinner();
```

Comment:
This needs to be changed to `tfm.tools.showSpinner()` since it was moved into [Webpack module](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_tools.js#L1)